Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/orientation.h

```diff
@@ -334,6 +350,91 @@ void reverse_face_orientations(const FaceRange& face_range, PolygonMesh& pmesh)
     }
 }
 
+/**
+* \ingroup PMP_orientation_grp
+* makes the orientation of all the connected_components of a `TriangleMesh` positive.
+* A closed polygon mesh is considered to have a positive orientation if the normal vectors
+* to all its faces point outside the domain bounded by the polygon mesh.
+* The normal vector to each face is chosen pointing on the side of the face
+* where its sequence of vertices is seen counterclockwise.
+*
+* @tparam TriangleMesh a model of `FaceListGraph` and `MutableFaceGraph` .
+* @tparam NamedParameters a sequence of \ref namedparameters
+*
+* @param tm a triangulated `TriangleMesh`
+* @param np optional sequence of \ref namedparameters among the ones listed below
+*
+* \cgalNamedParamsBegin
+*   \cgalParamBegin{vertex_point_map}
+*     the property map with the points associated to the vertices of `tm`.
+*     If this parameter is omitted, an internal property map for
+*     `CGAL::vertex_point_t` should be available in `TriangleMesh`
+*   \cgalParamEnd
+*   \cgalParamBegin{face_index_map}
+*     a property map containing the index of each face of `tm`.
+*   \cgalParamEnd
+* \cgalNamedParamsEnd
+*/
+template<class TriangleMesh, class NamedParameters>
+void orient_connected_components(TriangleMesh& tm, const NamedParameters& np)
+{
+  typedef boost::graph_traits<TriangleMesh> GT;
```

Comment:
`GT` is not a very suitable `typedef`... it is usually used for `Geom_traits`.
I would suggest `Graph_traits`, for example, to make sure it is not confusing